cactusbot/services/beam/parser.py

```diff
@@ -85,7 +85,7 @@ def parse_subscribe(cls, packet):
     def parse_resubscribe(cls, packet):
         """Parse resubscribe packet."""
 
-        return EventPacket("subscribe", packet["user"]["username"],
+        return EventPacket("resubscribe", packet["user"]["username"],
```

Comment:
This should stay as `subscribe`, just with a higher `streak` than `1`. :slightly_smiling_face: